README.md

```diff
@@ -1,6 +1,31 @@
-# 10x
+# 10x website Problem Statement and Goals
+
+### [Macro]
+
+**Opportunity:** The US Government employs many capable public servants who observe problems with government technology and are ideally positioned to pursue solutions to these problems. 
```

Comment:
replace`US Government` with `U.S. government` per https://content-guide.18f.gov/specific-words-and-phrases/